setup-scripts/README.md

```diff
@@ -11,40 +11,39 @@ that we are working with a Debian Buster Minimal system or similar.
 You may want to fork this repository and add your ssh pubkey to
 `authorized_keys` and optionally edit the `tmux` and `bash` configurations.
 
-NOTE: nginx version 1.11.0 or higher is required.
-If you use the install script below, the correct version should be installed.
+### Step 1: setting up server user
 
-0. SSH in a freshly installed Debian machine.
-1. `apt-get update && apt-get install sudo`
-1. `adduser user`
-1. `usermod -a -G sudo user`
-1. Quit the ssh session.
+1. SSH in a freshly installed Debian machine.
+2. `apt-get update && apt-get install sudo`
+3. `adduser user`
+4. `usermod -a -G sudo user`
+5. `usermod -a -G systemd-journal user`
+6. Quit the ssh session.
 
 You a can now ssh into your machine as the user `user`.
 
-5. On your local machine: `ssh-copy-id user@ip-addr`
-6. On your local machine: `ssh user@ip-addr`
-7. Now logged in as `user`: `sudo apt-get install git`
-8. `git clone https://github.com/NebulousLabs/skynet-webportal`
-9. `cd skynet-webportal/setup-scripts`
-10. `./setup.sh`
-11. Once DNS records are set you can run: `./letsencrypt-setup.sh`
-12. This should edit your nginx configuration for you. If not, you should check
-    that keys were created by letsencrypt in `/etc/letsencrypt/live/` and add
-    the following lines into your nginx configuration. Make sure to replace
-    `YOUR-DOMAIN` with your domain name.
-    ```
-    ssl_certificate /etc/letsencrypt/live/YOUR-DOMAIN/fullchain.pem;
-    ssl_certificate_key /etc/letsencrypt/live/YOUR-DOMAIN/privkey.pem;
-    ```
-13. Finally make sure to check your nginx conf and reload nginx:
-    `sudo nginx -t`
-    `sudo systemctl reload nginx`
+### Step 2: setting up environment
+
+1. On your local machine: `ssh-copy-id user@ip-addr`
+2. On your local machine: `ssh user@ip-addr`
+3. Now logged in as `user`
+
+**Following step will be executed on remote host logged in as a `user`:**
+
+1. `sudo apt-get install git`
+2. `git clone https://github.com/NebulousLabs/skynet-webportal`
+3. `cd skynet-webportal`
+4. edit `docker/nginx/conf.d/client.conf` line `server_name siasky.dev;` and replace `siasky.dev` with your domain name - it will be required for the SSL to generate successfully
+5. run setup scripts in the exact order, if one of them fails, you can retry just this one before proceesing
+   1. `setup-scripts/setup-server.sh`
+   2. `setup-scripts/setup-siad.sh`
+   3. `setup-scripts/setup-client.sh`
+   4. `setup-scripts/setup-health-check-scripts.sh`
 
 ## Running siad
 
 NOTE: You must be running `siad` and `siac` by building from a version at least
-as recent as `v1.4.4`.
+as recent as `v1.4.8`.
```

Comment:
1.4.10 broke some interoperability with older versions. Should we bump this to 1.4.11 or is it OK like this?

Reply:
```suggestion
as recent as `v1.4.11`.
```